src/iOS/toga_iOS/window.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, view):
     def statusbar_height(self):
         if UIApplication.sharedApplication.statusBarOrientation == UIInterfaceOrientation.Portrait.value:
             # This is the height of the status bar.
-            return 16
+            return UIApplication.sharedApplication.statusBarFrame.size.height
```

Comment:
At column 20: (F405) 'UIApplication' may be undefined, or defined from star imports: .libs